terminal/main.go

```diff
@@ -0,0 +1,21 @@
+package terminal
```

Comment:
Just some convenience code

Reply:
I guess this just has some argument utility functions?  Could we rename this to file to `arg_util.go` or `arg-util.go` or something else just to make the filename more meaningful instead of just `main.go` or would that not work?

Reply:
That'll work, I was just leaving it as simple as possible for now since there isn't much code in this package yet. So I would usually wait to split it up into separate files once there's a couple different types of functionality in there (at least in a language that's unopinionated about code pathing/organizing like Go). I can go either way on it though.

Reply:
Well in that case I guess we can just leave it for now until there are a couple of different types of functionality in there and can then split it up into separate files.